etc/config/openclc.defaults.properties

```diff
@@ -50,3 +50,13 @@ compiler.spirvclangdefaultspir.options=-finclude-default-header -fdeclare-opencl
 compiler.spirvclangdefaultspir64.exe=/usr/bin/clang
 compiler.spirvclangdefaultspir64.name=clang default (SPIR-V asm, spir64 triple)
 compiler.spirvclangdefaultspir64.options=-finclude-default-header -fdeclare-opencl-builtins -triple spir64-unknown-unknown
+
+group.clspv.compilers=clspv:clvk
+group.clspv.compilerType=clspv
+
+compiler.clspv.exe=/opt/compiler-explorer/build/bin/clspv
```

Comment:
Sorry, in the default config, paths should be where the tool is commonly installed, so probably `/usr/bin/` as you had initially? Provided the tool is still a prototype, anything will do, so you can pick what's best for you :)

You'll need to provide a `.amazon.properties` with very similar content, but this one will be used for the instance running godbolt.org. In this case `/opt/compiler-explorer/clspv-someversion/bin/clspv`.

After we have a matching build script in the `misc-builder` repo, we will install the compiler there :) Sorry about the previous confusion.